app/views/layouts/angular/_form_button_submit_angular.html.haml

```diff
@@ -3,16 +3,10 @@
 - confirm ||= ""
 - ng_show ||= true
 
-%div{"ng-show" => ng_show, "ng-controller" => "buttonGroupController"}
-  = button_tag(_("Submit"),
-                 :class     => "btn btn-primary disabled",
-                 :alt       => submit_title_off,
-                 :title     => submit_title_off,
-                 "ng-show"  => "!saveable(angularForm)",
-                 "ng-click" => "disabledClick($event)")
-  = button_tag(_("Submit"),
-                 :class     => "btn btn-primary",
-                 :alt       => submit_title_on,
-                 :title     => submit_title_on,
-                 "ng-show"  => "saveable(angularForm)",
-                 "ng-click" => "submitButtonClicked('#{confirm}')")
+%div{"ng-show" => ng_show}
+  %miq-button{:name            => _("Submit"),
+              "disabled-title" => submit_title_off,
+              "enabled-title"  => submit_title_on,
+              :enabled         => "saveable(angularForm)",
```

Comment:
Still using `saveable` so I'm guessing you didn't want to remove that `ng-controller` just yet? ;)

(`'diagnosticsDatabaseFormController` doesn't seem to set `$scope.saveable` anywhere...)